### ISSUE TITLE
Button should not contain link to server on default

### ISSUE THREAD

**Issue by erictbar** [kennethsible/jellyfin-rpc#1]
Thanks for this! The alternative has not been working for awhile.

I noticed that when I was logged in with my alt that there is a button `Watch in Jellyfin` that contains a link to the server. Since many people run personal/private servers, this can be concerning. I propose that this button be turned off (or alternativly, make the link just go to jellyfin.org or this repo) and, if specifcied in the .ini file, the URL to the server can be displayed.

**Comment by kennethsible** [comment]
I agree that the server address for Jellyfin should not be displayed on Discord, an oversight on my part. I have disabled the RPC button for "Play on Jellyfin" in 7db606d and pushed a [v1.2.4 release](https://github.com/kennethsible/jellyfin-rpc/releases/tag/v1.2.4). I may revisit this in the future and add an option to the INI as you suggested.
